README.md

```diff
@@ -129,8 +129,13 @@ To run tests on browser:
   `$ npm test`  
 
 To run tests on terminal:  
+  
+## ESLint  
+  You can install the ESLint extension at VSCode editor or you can run the `./node_modules/.bin/eslint enrollment.js cypress/ gulpfile.js` task.  
```

Comment:
can you create a "eslint" gulp task, that run `./node_modules/.bin/eslint enrollment.js cypress/ gulpfile.js` and update the command here for just `gulp eslint`?

Reply:
Of course, I made the gulp task in this commit